attachments/views.py

```diff
@@ -30,6 +33,12 @@ def attach(request, session_id):
         content_type = 'text/plain' if request.POST.get('X-Requested-With', '') == 'IFrame' else 'application/json'
         try:
             f = request.FILES['attachment']
+            if getattr(settings, 'ATTACHMENTS_MAXIMUM_FILE_SIZE', False):
```

Comment:
Nitpicky, but you can just grab the setting once with getattr, instead of every time down below. It's less maintenance if we ever need to change a default value or something, and just shorter lines.